include/modem/pdn.h

```diff
@@ -243,7 +243,7 @@ __deprecated int pdn_dynamic_params_get(uint8_t cid, struct in_addr *dns4_pri,
  *
  * @return Zero on success or an error code on failure.
  */
-int pdn_dynamic_info_get(uint32_t cid, struct pdn_dynamic_info *pdn_info);
+int pdn_dynamic_info_get(uint8_t cid, struct pdn_dynamic_info *pdn_info);
```

Comment:
Why not add the family here as a parameter? As it is now, when this function returns `0` the user does not know whether  a valid IPv4 address was obtained, or a valid IPv6 address was obtained or both. The only option is to call another function on each `in_addr` structure and verify if it contains a valid address.

By adding one more parameter the function can then return an error if the given CID does not support that address family, and so the user doesn't have to check the addresses manually.

Reply:
I see, then I guess this is fine.